db/migrate/20230619204213_add_trainer_reference_to_team.rb

```diff
@@ -0,0 +1,5 @@
+class AddTrainerReferenceToTeam < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :teams, :trainers, null: false, foreign_key: true
```

Comment:
You might need to change this to `:trainer` in order to get `trainer_id` instead of `trainers_id` but I'm not sure exactly how `add_reference` works.

You could also just remove this and add `t.trainer_id`  column and foreing key in your `CreateTeams` migration.

Reply:
I fixed this by running a new migration with the change_column method on the teams table.